types/remarkable/lib/index.d.ts

```diff
@@ -374,97 +385,446 @@ declare namespace Remarkable {
          * The content of the block. This might include inline mardown syntax
          * which may need further processing by the inline rules.
          */
-        content: string;
+        content?: string;
 
         /**
          * This is initialized with an empty array (`[]`) and will be filled
          * with the inline parser tokens as the inline parsing rules are applied.
          */
-        children: Token[];
+        children?: Token[];
     }
 
     interface ContentToken extends TagToken {
         /**
          * A text token has a `content` property. This is passed to
          * the corresponding renderer to be converted for output.
          */
-        content: any;
+        content?: any;
 
         /**
-         * The type of the token.
+         * Is this a block element
          */
-        type: string;
+        block?: boolean;
+    }
 
+    // ---------------
+    // Specific Block Tokens
+    // ---------------
+
+    interface BlockquoteToken extends TagToken {}
```

Comment:
Why use an empty interface? Ins't better instead for the interfaces to extend `TagToken` so that `BlockquoteToken` serves no purpose?

Same question for a couple more empty interfaces.

Reply:
I kept like this for functional purposes. I wanted for every interfaces to reflect its source to prevent any evolution.  
I agree with your proposal ; I just suggested a costless preventing.

Reply:
Sounds good, thanks for clarifying.